### PR TITLE
[RDAPI-26817] - Mount Persistent YAML Volume as Directory in EMT

### PR DESCRIPTION
**Pull request by dshielaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Describe how to mount a persistent volume using a YAML Directory in EMT

## Deploy preview link

https://deploy-preview-2283--axway-open-docs.netlify.app/docs/apim_howto_guides/configuring_apigw_container/index.html

## Checklist for contributors

Before submitting this PR, please make sure:

_This is simply a reminder of what we are going to look for before merging your change._

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

### CODE REVIEW

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ docker run -d --name=apimgr --network=api-gateway-domain \
            api-gateway-my-group:1.0
 ```
 
-### YAML Entity Store configuration
+### YAML Entity Store configuration (tar.gz)
```
Comment (andreamussap) [inline content/en/docs/apim_howto_guides/configuring_apigw_container.md:101]: @dshielaxway please change this to one section with 2 sub-sections, e.g.

## YAML Entity Store configuration

YAML based API Gateway policy configuration can be stored both as a deployment package (.tar.gz file) or as a directory. The following examples show to add them to the Docker container runtime configuration

### Deployment package

For configuration stored in a deployment package (.tar.gz file), add the source directory to the /merge/yaml Docker volume:

// code

### Directory

For configuration stored in a directory, add the source directory to the /merge/yaml Docker volume:

// code

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ To create your persisted API Gateway store, the expected mounted directory (moun
 | Docker volume mount | Description |
 | ------------------- | ----------- |
 | /merge/fed | For an API Gateway policy configuration stored as a deployment package (.fed file).|
-| /merge/yaml | For YAML based API Gateway policy configuration stored as a deployment package (.tar.gz file).|
+| /merge/yaml | For YAML based API Gateway policy configuration stored either as a deployment package (.tar.gz file), or as a directory.|
```
Comment (andreamussap) [inline content/en/docs/apim_howto_guides/configuring_apigw_container.md:28]: ((.tar.gz file) , or) - you can delete the comma

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md (hunk 1 above)
Comment (dshielaxway) [reply content/en/docs/apim_howto_guides/configuring_apigw_container.md]: Done, thanks

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md (hunk 2 above)
Comment (dshielaxway) [reply content/en/docs/apim_howto_guides/configuring_apigw_container.md]: Done, thanks

File: content/en/docs/apim_howto_guides/configuring_apigw_container.md (hunk 1 above)
Comment (dshielaxway) [reply content/en/docs/apim_howto_guides/configuring_apigw_container.md]: Apologies @andreamussap,
I completely missed those descriptions. I've added them in, with some slight tweaks.
Cheers!

**Review by credmond-axway — APPROVED** [review]

**Pull request merged**